src/aboutsettings.cpp

```diff
@@ -39,18 +39,23 @@
 #include <QFile>
 #include <QByteArray>
 #include <QRegularExpression>
+#include "ssudeviceinfo.h"
 
 AboutSettings::AboutSettings(QObject *parent)
-    : QObject(parent),
-      m_sysinfo(new QStorageInfo(this)),
-      m_netinfo(new QNetworkInfo(this)),
-      m_devinfo(new QDeviceInfo(this))
+    : QObject(parent)
+    , m_sysinfo(new QStorageInfo(this))
+    , m_netinfo(new QNetworkInfo(this))
+    , m_devinfo(new QDeviceInfo(this))
+    , m_ssuinfo(new SsuDeviceInfo())
```

Comment:
Yes, for this one time it does ;) Next member variable addition will be a single-line diff, though :)